tests/OmniSharp.Roslyn.CSharp.Tests/CompletionFacts.cs

```diff
@@ -2250,6 +2250,24 @@ public C()
             }
         }
 
+        [Theory]
+        [InlineData("dummy.cs")]
+        [InlineData("dummy.csx")]
+        public async Task ReplacesUpUntilCursorInMiddleOfWord(string filename)
+        {
+            const string input =
+                @"pub$$class}";
```

Comment:
Based on the bug I created recently, we might want to have a multi-line code file and have the word not start at the beginning of the file for a more generalized test XD